src/main/java/org/oasis_eu/portal/core/model/catalog/CatalogEntry.java

```diff
@@ -86,6 +86,9 @@
     @JsonProperty("visible")
     private boolean visible = false;
 
+    @JsonProperty("visibility")
```

Comment:
Ouch, visibility and access_control modifications should be in another feature branch (this is not related to profile rewrite)